modules/travel_pay/app/controllers/travel_pay/claims_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class ClaimsController < ApplicationController
+    def index
+      veis_token = client.request_veis_token
+
+      # Non-intuitive Ruby behavior: #split splits a string on space by default
+      vagov_token = request.headers['Authorization'].split[1]
+      btsss_token = client.request_btsss_token(veis_token, vagov_token)
+
+      begin
+        claims = client.get_claims(veis_token, btsss_token)
+      rescue Faraday::Error => e
+        raise common_exception(e)
+      end
+
+      render json: claims, each_serializer: TravelPay::ClaimSerializer, status: :ok
+    end
+
+    private
+    def client
+      @client ||= TravelPay::Client.new
+    end
+
+    def common_exception(e)
```

Comment:
I think I'd like to move this out of the controller and let the `Common::Exception` bubble up and out. We can't do this currently, but if we implement the changes described in department-of-veterans-affairs/va.gov-team#78913, it would be doable